### PR TITLE
Add BlockChain concept

### PR DESCRIPTION
**Pull request by jadia**
This PR implements the Blockchain concept in a primitive manner. The name of blockchain is `darkCoin`.

The script can be tested by running using python3

```bash
python3 darkCoin.py
```

### CODE REVIEW

File: Code-Sleep-Python/primitiveBlockChain/darkCoin.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    def __init__(self, index, timestamp, data, previous_hash):
+        self.index = index
+        self.timestamp = timestamp
+        self.data = data
+        self.previous_hash = previous_hash
+        self.hash = self.hashBlock()
+
+# Create hash of block
+    def hashBlock(self):
+        sha = hasher.sha256()
```
Comment (prateekiiest) [inline Code-Sleep-Python/primitiveBlockChain/darkCoin.py]: is the original version of blockchain you are implementing?
Can you add an application-specific one here ?

You may get an idea from one of my projects : https://github.com/prateekiiest/Anomaly-Detection-in-BlockChain

**Comment by prateekiiest** [comment]
Could you update on this please?

File: Code-Sleep-Python/primitiveBlockChain/darkCoin.py (hunk 1 above)
Comment (jadia) [reply Code-Sleep-Python/primitiveBlockChain/darkCoin.py]: Few things are missing such as nounce, etc. I tried to make a _chain_ with the block of information. I'll improve it in some time.

Comment (prateekiiest) [reply Code-Sleep-Python/primitiveBlockChain/darkCoin.py]: sure @jadia, this one is promising. Let me know if any updates.

**Comment by prateekiiest** [comment]
\stale dont mark it stale

**Comment by prateekiiest** [comment]
I am adding this in @jadia . Let me know if you come up with an improved version.

**Pull request merged**